docs/ARCHITECTURE.md

```diff
@@ -1,7 +1,7 @@
 ## Architecture
 
 ### High-level overview
-![Draft Architecture](https://github.com/18F/identity-idp/raw/amos/feature/arch_docs_r1/docs/diagrams/draft-architecture-overview.png)
+![Draft architecture](https://raw.githubusercontent.com/18F/identity-idp/master/docs/diagrams/draft-architecture-overview.png)
```

Comment:
Check it out! I pushed a560ba7b3a4462061575dd5d90f646e638cfa66a and then clicked the "view" button at the top of the diff for this file. Images load!

https://github.com/18F/identity-idp/blob/a560ba7b3a4462061575dd5d90f646e638cfa66a/docs/ARCHITECTURE.md